doc/source/examples/index.rst

```diff
@@ -17,23 +28,19 @@ Here are a series of examples using MAPDL with the ``ansys-mapdl-core`` library.
    :start-line: 2
 
 
-.. === PYMAPDL EXAMPLES ===
+.. === Dev portal examples ==
```

Comment:
```suggestion
.. === Developer Portal examples ==
```
To match the capitalization of the name used earlier?

Reply:
those are comments... but yeah.